### ISSUE TITLE
I can't run any software through Norton Commander

### ISSUE THREAD

**Issue by BridgeHeadland** [joncampbell123/dosbox-x#3759]
### Describe the bug

I'm using Norton Commander version 5.51 (the latest) in DOSBox-X, but in the latest versions - including Development Builds, I can't launch any software from NC. Instead, NC quiting. The same thing happens when I type a command line and press enter. I'm just saying.
It's not the first time it's happened, I know, so this thread should never be closed, in case it happens again.

### Steps to reproduce the behaviour

1. Scroll down to an executable file.
2. Start the executable file.
3. You will see Norton Commander quit.

### Expected behavior

I expect the executable to start, while in Norton Commander.

### What operating system(s) this bug have occurred on?

Any Windows 10

### What version(s) of DOSBox-X have this bug?

The few latest versions, I don't remember which version started this.

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by scrutinizer80** [comment]
It's working fine here...

**Comment by BridgeHeadland** [comment]
@scrutinizer80 Oh? What version of DOSBox-X are you using? I'm not sure we're using the same version.

**Comment by scrutinizer80** [comment]
> Oh? What version of DOSBox-X are you using? I'm not sure we're using the same version.

2022.09.0 (0.84.3) SDL2 on Windows. The MinGW build.

**Comment by NebularNerd** [comment]
After some testing I suspect that @BridgeHeadland may have Long Filenames enabled. In my tests it appears that enabling LFN support breaks NC giving the issues described. 5.51 does sort of support LFN but only with an external TSR program [Norton Commander article, see section 1.1](https://p2k.utn.ac.id/IT/en/3069-2966/Norton-Commander_20498_p2k-utn.html)  

From my testing NC 5.51 behaves as follows (it's sufficient to exit NC before testing each option, no need to restart DosBox):
- DosBox DOS 5.00 (default): Works as expected, everything works as it should
- DosBox DOS 6.22: As above
- DosBox DOS 7.10: NC Breaks (NOTE: setting to 7.10 automatically enables LFN as it replicates a Win95'ish DOS)
- DosBox DOS 7.10 (Long filename disabled): Works as expected
- DosBox DOS 5.00 (Long filename enabled): NC Breaks
- MS-DOS 6.22 HD Image: Works as expected

Using [DOSLFN from here](https://sta.c64.org/lfnemu.html) and running `DOSLFN` before starting NC cures the issue when LFN is enabled. This is not a DosBox issue by the looks of it, more a limitation of how NC interacts with the underlying DOS environment, you would likely experience similar trying to use it on a real system running DOS 7 or FreeDOS with LFN support.

**TL;DR**
Run NC under DOS 5, or use DOSLFN under 7.10/LFN enabled setups.

**Comment by NebularNerd** [comment]
@BridgeHeadland did this work out for you?

**Comment by Nafets44** [comment]
It's correct, when I switch off lfn the NC works, when I switch on lfn the NC doesn't work. No matter which DOS version. DOSLFN doesn't work either. No matter which version of DOSLFN I load before the NC, the NC does not show long file names. But vDosPlus works very well with the NC and long file names, why doesn't DosBox-X?
